css-canvas-medical/styles.css

```diff
@@ -0,0 +1,278 @@
+.container {
+  width: 75%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.column-half {
+  display: flex;
```

Comment:
Instead of putting `display: flex` on a column, you would be better off leaving this with just `flex-basis`, and make the first child of the column element a `div` with the `row` class, so you can make a new row with flexbox capabilities.